docs/UserGuide.md

```diff
@@ -174,7 +178,7 @@ Example:
 
 #### Adding a lesson: `add lesson`
 
-Adds a lesson to the ModBook. This command can be executed in the **details** view.
+Adds a lesson to the ModBook. This command can be executed in the [details view](#details-view).
```

Comment:
make it consistent, change it to "This command can only be executed in the details view" (you're missing an "only")

Reply:
same with line 229